src/config/default.json

```diff
@@ -0,0 +1,4 @@
+{
+  "databaseConnectionString": "postgres://localhost:5432/node-js-epam-mentoring",
```

Comment:
it is better to use env file and sure never store credentials in repo)))

Reply:
well thank you :)